hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/multimap/AbstractMultiMapAddEntryListenerMessageTask.java

```diff
@@ -102,12 +101,12 @@ public void onMapEvent(MapEvent event) {
                 final EntryEventType type = event.getEventType();
                 final String uuid = event.getMember().getUuid();
                 sendClientMessage(null, encodeEvent(null,
-                        null, type.getType(),
+                        null, null, type.getType(),
                         uuid, event.getNumberOfEntriesAffected()));
             }
         }
     }
 
-    protected abstract ClientMessage encodeEvent(Data key, Data value,
+    protected abstract ClientMessage encodeEvent(Data key, Data value, Data oldValue,
```

Comment:
Well, your proposed ordering is more convenient to me but the convention on event handling for map and multimap uses (value, oldValue) order, so I leave this as is since it can lead to mistakes if I just change this.